lagtraj/forcings/profile_calculation.py

```diff
@@ -225,6 +239,18 @@ def calculate_timestep(ds_profile_posn, ds_domain, sampling_method):
     # required when calculating the representative vertical profile and
     # horizontal gradients
     ds_ref_pt = ds_profile_posn[["lat", "lon", "time"]]
+
+    ds_subdomain = ds_subdomain.where(True, other=np.nan)
```

Comment:
It's funny that this is needed. Maybe add a comment that we don't know why? Also I think a comment for the fact that we have to do the `local` sampling before we apply the mask would be great, just so it's clear why there are two loops

Reply:
It is not an ideal solution, but indeed good to add some more documentation. Will do.

Reply:
Great